Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -950,433 +906,440 @@
+    <RegEx find="([gG])rj?eši(o|m|š|mo|te|la|li|ti)" replaceWith="$1riješi$2" />
+    <RegEx find="haos" replaceWith="kaos" />
+    <RegEx find="hemij" replaceWith="kemij" />
+    <RegEx find="Hemij" replaceWith="Kemij" />
+    <RegEx find="holesterol" replaceWith="kolesterol" />
+    <RegEx find="Holesterol" replaceWith="Kolesterol" />
+    <RegEx find="\bHrist(a|e|u|om)?\b" replaceWith="Krist$1" />
+    <RegEx find="hrišćan" replaceWith="kršćan" />
+    <RegEx find="Hrišćan" replaceWith="Kršćan" />
+    <RegEx find="\bHoland(ska|sku|skom|skoj)\b" replaceWith="Nizozems$1" />
+    <RegEx find="\bNizuzem(ska|sku|skom|skoj)\b" replaceWith="Nizozems$1" />
+    <RegEx find="\bhistorijsk(e|i|a|o|u|om|oj)\b" replaceWith="povijesn$1" />
+    <RegEx find="\bHistorijsk(e|i|a|o|u|om|oj)\b" replaceWith="Povijesn$1" />
+    <RegEx find="\bistorijsk(e|i|a|o|u|om|oj)?\b" replaceWith="povijesn$1" />
+    <RegEx find="\bIstorijsk(e|i|a|o|u|om|oj)?\b" replaceWith="Povijesn$1" />
+    <RegEx find="\bhiljad(a|e|i|u|om|ama)\b" replaceWith="tisuć$1" />
+    <RegEx find="hleb" replaceWith="kruh" />
+    <RegEx find="hroničn" replaceWith="kroničn" />
+    <RegEx find="Hroničn" replaceWith="Kroničn" />
+    <RegEx find="([iI])dentifikuje(m|š|mo|te|ju|še)" replaceWith="$1dentificira$2" />
+    <RegEx find="([iI])migracion(a|i|u|e|om|og)?" replaceWith="$1migracijsk$2" />
+    <RegEx find="migracion(a|i|u|e|om|og)?" replaceWith="migracijsk$1" />
+    <RegEx find="([iI])nostranstv(a|u|o|om|ima)" replaceWith="$1nozemstv$2" />
+    <RegEx find="interesantn" replaceWith="zanimljiv" />
+    <RegEx find="Interesantn" replaceWith="Zanimljiv" />
+    <RegEx find="([iI])spoljava" replaceWith="$1zražava" />
+    <RegEx find="([iI])zbe(ći|gava|gavaš|gavamo|gavate|gavaju|gavanje|gavali|gao|gla|gli|glo)" replaceWith="$1zbje$2" />
+    <RegEx find="([iIz])blj?ed(ela|elo|ele|eli|io|elom|elima|jele)" replaceWith="$1blijedj$2" />
+    <RegEx find="([iI])sčeznu(o|la|le|li|lu|lom|lima|lima|ti|uše|uvši)" replaceWith="$1ščeznu$2" />
+    <RegEx find="([iI])sčezn(em|eš|e|emo|ete|u)" replaceWith="$1ščezn$2" />
+    <RegEx find="([iI])skorišć" replaceWith="$1skorišt" />
+    <RegEx find="([iI])zmen(a|e|i|u|om|ama)" replaceWith="$1zmjen$2" />
+    <RegEx find="ispresecan(a|e|i|u|om)?" replaceWith="ispresijecan$1" />
+    <RegEx find="([iI])zgladne(la|le|li|lo|lu|lima|lom|vši)" replaceWith="$1zgladnje$2" />
+    <RegEx find="([iI])zvešta(j|ji|ja|je|ju|vam|vaš|va|vamo|vate|vaju|vanje)?" replaceWith="$1zvješta$2" />
+    <RegEx find="\b([jJ])agnje" replaceWith="$1anje" />
+    <RegEx find="\bJovan(a|e|u|om)?" replaceWith="Ivan$1" />
+    <RegEx find="\b([kK])af(a|e|i|u|om|ama|ana|ane|ani|anu|anom|anama|anska|anski|ansku|anskom)\b" replaceWith="$1av$2" />
+    <RegEx find="kancelarij(u|om)?" replaceWith="ured$1" />
+    <RegEx find="kašik(a|e|i|o|u|om|ama)" replaceWith="žlic$1" />
+    <RegEx find="ćerk(e|i|o|om)?" replaceWith="ćeri$1" />
```

Comment:
NO, because kćer is singular, kćeri is plural.

however, what you could do is add čČ in this example, because č and ć are obviously very similar letters and  kčerk(e|i|o|om) is, obviously, wrong - so it could fix that also.